lib/utils/cli_test.go

```diff
@@ -116,19 +116,19 @@ func TestAllowNewlines(t *testing.T) {
 		},
```

Comment:
```suggestion
		},
		{   
                    in:  "\thello, world!",
                    out: "\thello, world!",
                },  
                {   
                    in:  "\t\thello, world!",
                    out: "\t\thello, world!",
                },  
                {   
                    in:  "hello, world!\n",
                    out: "hello, world!\n",
                },  
                {   
                    in:  "hello, world!\n\n",
                    out: "hello, world!\n\n",
                },  
                {   
                    in:  string([]byte{0x68, 0x00, 0x68}),
                    out: "\"h\\x00h\"",
                },  
                {   
                    in:  string([]byte{0x68, 0x08, 0x68}),
                    out: "\"h\\bh\"",
                },  
                {   
                    in:  string([]int32{0x00000008, 0x00000009, 0x00000068}),
                    out: "\"\\b\"\th",
                },  
                {   
                    in:  string([]int32{0x00000090}),
                    out: "\"\\u0090\"",
                },
```

Some more test cases to consider, these all pass with the current implementation.

Reply:
Thanks for the test cases!